.github/workflows/build.yml

```diff
@@ -230,11 +231,117 @@ jobs:
+
+        - name: Cache-op for build-cache through ccache
+          uses: actions/cache@v2
+          with:
+            path: |
+                ${{ env.ccache_dir }}
+                ${{ github.workspace }}/emsdk/ccache/git-emscripten_64bit/
+            key: ccache-${{ github.job }}-${{ env.emsdk_version }}-${{ steps.ccache_vars.outputs.hash }}-${{ github.ref }}-${{ steps.ccache_vars.outputs.timestamp }}
+            restore-keys: |-
+              ccache-${{ github.job }}-${{ env.emsdk_version }}-${{ steps.ccache_vars.outputs.hash }}-${{ github.ref }}
+              ccache-${{ github.job }}-${{ env.emsdk_version }}-${{ steps.ccache_vars.outputs.hash }}
+              ccache-${{ github.job }}-${{ env.emsdk_version }}
+
+        - name: Setup Emscripten toolchain
+          run: |
+              (cd emsdk && ./emsdk install ${{ env.emsdk_version }} ccache-git-emscripten-64bit)
+              (cd emsdk && ./emsdk activate ${{ env.emsdk_version }} ccache-git-emscripten-64bit)
+              # mtime of this file is checked by ccache, we set it to avoid cache misses.
+              touch -m -d '1 Jan 2021 12:00' emsdk/.emscripten
+
+              # These needs to be done in the activated shell.
+              eval $(./emsdk/emsdk construct_env \
+                  | sed 's/export PATH=\(.*\);/echo \1 >> $GITHUB_PATH;/' \
+                  | sed 's/export \(.*\);/echo \1 >> $GITHUB_ENV;/' );
+
+              # This looks more permanent than version pinned, so keeping temporarily to avoid failures.
+              echo "${{ github.workspace }}/emsdk/ccache/git-emscripten_64bit/bin" >> $GITHUB_PATH
+
+        - name: Generate ccache_vars for ccache based on machine
+          shell: bash
+          id: ccache_vars
+          run: |-
+            echo "::set-output name=hash::$(echo ${{ env.ccache_compilercheck }})"
+            echo "::set-output name=timestamp::$(date '+%Y-%m-%dT%H.%M.%S')"
+
+        - name: Verify Emscripten setup
+          run: |
+              emcc --version
+              emcmake cmake --version
+              emmake make --version
+
+        - name: Configure builds
+          run: |
+            mkdir -p build-wasm
+            cd build-wasm
+            emcmake cmake -DCOMPILE_WASM=on ..
+
+        - name: ccache prolog
+          run: |-
+            ccache -s # Print current cache stats
+            ccache -z # Zero cache entry
+
+        - name: Compile
+          working-directory: build-wasm
+          run: |
+            emmake make -j2
+
+        - name: ccache epilog
+          run: |
+            ccache -s # Print current cache stats
+
+        - name: Instantiate simd wormhole
+          working-directory: build-wasm
+          run: bash ../wasm/patch-artifacts-enable-wormhole.sh
+
+        - name: Import GEMM library from a separate wasm module
+          working-directory: build-wasm
+          run: bash ../wasm/patch-artifacts-import-gemm-module.sh
```

Comment:
It appears the patch is done here but is not enough. 

[CircleCI](https://github.com/browsermt/bergamot-translator/blob/main/.circleci/config.yml) appears to be doing two separate builds. Unsure what exactly is going on. I can't get the built js/wasm to work with chrome but it appears to be working in Firefox (Nightly).